bench/src/lib.rs

```diff
@@ -199,36 +200,51 @@ pub fn create_memo_tx(
     blockhash: Hash,
     rng: &mut Rng8,
     size: tx_size::TxSize,
+    cu_price_micro_lamports: u64,
 ) -> Transaction {
     let rand_str = generate_random_string(rng, size.memo_size());
 
     match size {
-        tx_size::TxSize::Small => create_memo_tx_small(&rand_str, payer, blockhash),
-        tx_size::TxSize::Large => create_memo_tx_large(&rand_str, payer, blockhash),
+        tx_size::TxSize::Small => {
+            create_memo_tx_small(&rand_str, payer, blockhash, cu_price_micro_lamports)
+        }
+        tx_size::TxSize::Large => {
+            create_memo_tx_large(&rand_str, payer, blockhash, cu_price_micro_lamports)
+        }
     }
 }
 
-pub fn create_memo_tx_small(msg: &[u8], payer: &Keypair, blockhash: Hash) -> Transaction {
+pub fn create_memo_tx_small(
+    msg: &[u8],
+    payer: &Keypair,
+    blockhash: Hash,
+    cu_price_micro_lamports: u64,
+) -> Transaction {
     let memo = Pubkey::from_str(MEMO_PROGRAM_ID).unwrap();
 
-    // TODO make configurable
-    // 3 -> 6 slots
-    // 1 -> 31 slots
-    let cu_budget: Instruction = ComputeBudgetInstruction::set_compute_unit_price(3);
+    let cu_budget_ix: Instruction =
+        ComputeBudgetInstruction::set_compute_unit_price(cu_price_micro_lamports);
     // Program consumed: 12775 of 13700 compute units
-    let priority_fees: Instruction = ComputeBudgetInstruction::set_compute_unit_limit(14000);
+    let cu_limit_ix: Instruction = ComputeBudgetInstruction::set_compute_unit_limit(14000);
```

Comment:
could you port the cu_requested over from https://github.com/blockworks-foundation/lite-rpc/pull/363/commits/73d8823b28a684fecad2be1b8b41d60614fc9b83